src/vehicle_use.cpp

```diff
@@ -1149,7 +1149,7 @@ void vehicle::operate_scoop()
                 that_item_there->inc_damage();
                 //The scoop gets a lot louder when breaking an item.
                 sounds::sound( position, rng( 10,
-                                              that_item_there->volume() * 2 / units::legacy_volume_factor + 10 ),
+                                              that_item_there->volume() * 2 / 260_ml ),
```

Comment:
```suggestion
                                              that_item_there->volume() * 2 / 250_ml + 10 ),
```

`*` and `/` have a priority before `+`